osc_sdk/sdk.py

```diff
@@ -532,10 +533,17 @@ def build_headers(self, target, json_parameters):
 
 
 def get_conf(profile):
-    conf_path = pathlib.Path.home() / CONFIGURATION_FOLDER / CONFIGURATION_FILE
-    if not conf_path.exists():
+    conf_paths = {
+        pathlib.Path.home() / OLD_CONFIGURATION_FOLDER / CONFIGURATION_FILE,
+        pathlib.Path.home() / CONFIGURATION_FOLDER / CONFIGURATION_FILE,
+    }
+    # Raise an error if neither of the two paths exists.
+    if not any(path.exists() for path in conf_paths):
         raise RuntimeError('No configuration file found in home folder')
 
+    # Check which conf_path is used.
+    conf_path = next(path for path in conf_paths if path.exists())
```

Comment:
next(..., None)

if not conf_path:
  raise ...

Reply:
ah yess, mis à jour !